api/tests/Api/CampCollaborations/ResendInvitationCampCollaborationTest.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace App\Tests\Api\CampCollaborations;
+
+use App\Entity\CampCollaboration;
+use App\Tests\Api\ECampApiTestCase;
+
+/**
+ * @internal
+ */
+class ResendInvitationCampCollaborationTest extends ECampApiTestCase {
```

Comment:
Allenfalls noch ein Test dass Gast-Benutzer das nicht dürfen, und möglicherweise separate Tests für Manager und Member, falls wir denen in Zukunft mal separate Berechtigungen geben.

Reply:
Aber sicher, habe ich hinzugefügt.